src/cosmos/useProvider.ts

```
@@ -2,21 +2,48 @@ import { initializeConnector } from '@web3-react/core'
 import { MetaMask } from '@web3-react/metamask'
 import { Connector } from '@web3-react/types'
 import { WalletConnect } from '@web3-react/walletconnect'
+import { SupportedChainId } from 'constants/chains'
 import { useEffect, useState } from 'react'
 
-import { INFURA_NETWORK_URLS } from './useJsonRpcEndpoint'
 import useOption from './useOption'
 
+const INFURA_KEY = process.env.INFURA_KEY
+if (INFURA_KEY === undefined) {
+  console.warn(`INFURA_KEY must be a defined environment variable to use jsonRpcUrlMap in the cosmos viewer`)
+}
+
+export const INFURA_NETWORK_URLS: { [chainId: number]: string[] } = INFURA_KEY
```

think we should return an error rather than warn here